contracts/governance/Vesting/VestingLogic.sol

```diff
@@ -110,68 +120,75 @@ contract VestingLogic is IVesting, VestingStorage, ApprovalReceiver {
     }
 
     /**
-     * @notice Withdraws all tokens from the staking contract and
+     * @notice Withdraws unlocked tokens from the staking contract and
      * forwards them to an address specified by the token owner.
-     * @param receiver The receiving address.
-     * @dev Can be called only by owner.
+     * @param _receiver The receiving address.
      * */
-    function governanceWithdrawTokens(address receiver) public {
-        require(msg.sender == address(staking), "unauthorized");
-
-        _withdrawTokens(receiver, true);
+    function withdrawTokens(address _receiver) public onlyOwners {
```

Comment:
```suggestion
      function withdrawTokens(address _receiver) external
```

Reply:
Fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/c4c43c22e2dc30a936960e01e9a8acd6e2053616)